pkg/test/util/file/file.go

```diff
@@ -128,3 +129,34 @@ func ReadTarFile(filePath string) (string, error) {
 	}
 	return "", fmt.Errorf("file not found %v", filePath)
 }
+
+// ReadDir returns the names of all files in the given directory. This is not recursive.
+// The base path is a appended; for example, ReadDir("dir") -> ["dir/file1", "dir/folder1"]
+func ReadDir(filePath string, extensions ...string) ([]string, error) {
+	dir, err := ioutil.ReadDir(filePath)
```

Comment:
A reason not to use os.ReadDir?

Reply:
isn't it the same?

Reply:
They are similar, but I recall that  package io and os implementations are preferred in new code (you may recall when we tried moving to them earlier that the build tooling).

Reply:
done. If we don't want ioutil used we should add a linter.